content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -68,11 +68,6 @@ kubectl config get-contexts                          # display list of contexts
 kubectl config current-context                       # display the current-context
 kubectl config use-context my-cluster-name           # set the default context to my-cluster-name
 
-kubectl config set-cluster my-cluster-name           # set a cluster entry in the kubeconfig
-
-# configure the URL to a proxy server to use for requests made by this client in the kubeconfig
-kubectl config set-cluster my-cluster-name --proxy-url=my-proxy-url
-
 # add a new user to your kubeconf that supports basic auth
 kubectl config set-credentials kubeuser/foo.kubernetes.com --username=kubeuser --password=kubepassword
```

Comment:
Why this PR reverting English changes?

Reply:
It was caused by my misoperation in rebase, maybe I should establish a PR to solve this problem